Furball.Vixie.TestApplication/Tests/TestTextureRenderTargets.cs

```diff
@@ -33,9 +33,9 @@ public class TestTextureRenderTargets : GameComponent {
             this._renderTarget.Bind();
             GraphicsBackend.Current.Clear();
 
-            this._lineRenderer.Begin();
-            this._lineRenderer.Draw(new Vector2(0, 0), new Vector2(1280, 720), 16f, Color.Red);
-            this._lineRenderer.End();
+            //this._lineRenderer.Begin();
+            //this._lineRenderer.Draw(new Vector2(0, 0), new Vector2(1280, 720), 16f, Color.Red);
+            //this._lineRenderer.End();
```

Comment:
Why was this commented out?

Reply:
that was back then when d3d11 couldnt do lines, will add back